packages/bsky/src/notifications.ts

```diff
@@ -0,0 +1,285 @@
+import axios from 'axios'
+import Database from './db'
+import { Notification } from './db/tables/notification'
+import { AppBskyEmbedImages, AtUri } from '@atproto/api'
+import { Insertable } from 'kysely'
+
+const PUSH_NOTIF_SERVER_BASE_URL = 'https://push.bsky.app'
+const PUSH_NOTIF_SERVER_ENDPOINT = '/api/push'
+export const PUSH_NOTIF_SERVER_URL =
+  PUSH_NOTIF_SERVER_BASE_URL + PUSH_NOTIF_SERVER_ENDPOINT
+export const BSKY_APP_ID = 'xyz.blueskyweb.app'
+type Platform = 'ios' | 'android' | 'web'
+type PushNotification = {
+  tokens: string[]
+  platform: 1 | 2 // 1 = ios, 2 = android
+  title: string
+  message: string
+  topic: string
+  data?: {
+    [key: string]: string
+  }
+}
+type InsertableNotif = Insertable<Notification>
+
+export class NotificationServer {
+  constructor(public db: Database) {}
+  static creator() {
```

Comment:
is this creator function being used anywhere?

Reply:
Nope, removed.